x-pack/plugins/search_indices/public/components/indices/indices_router.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { Route, Routes } from '@kbn/shared-ux-router';
+import { ROOT_PATH, SEARCH_INDICES_DETAILS_PATH } from '../../routes';
+import { SearchIndexDetailsPage } from './details_page';
+import { ElasticsearchStartPage } from '../start/start_page';
+
+export const SearchIndicesRouter: React.FC = () => {
+  return (
+    <Routes>
+      <Route exact path={SEARCH_INDICES_DETAILS_PATH} component={SearchIndexDetailsPage} />
+      <Route path={ROOT_PATH} component={ElasticsearchStartPage} />
```

Comment:
🤔  I was thinking we'd want to remove this, but the start page will redirect to index management if they have > 1 index, and if they have no indices it wouldn't be a bad place to send them.

But if thats the behavior we want I would say we should still make this a redirect to `/app/elasticsearch/start` and not a Route with the `ElasticsearchStartPage` component.

Reply:
No not within `SearchIndexDetailsPage`

I thinking more like:

```suggestion
      <Route exact path={ROOT_PATH}>
        <Redirect to={`<SOME_PATH>`} />
      </Route>
```

What I'm unsure about is if you can redirect to another kibana application using the `<Redirect>` component.

Reply:
It also supports using `<Redirect>` instead of `<Route>` 

```
<Redirect from={ROOT_PATH} to={OTHER_PATH} />
```

[Redirect docs](https://v5.reactrouter.com/web/api/Redirect)